kv/test_raftstore/test_test.go

```diff
@@ -702,3 +702,17 @@ func TestSplitConfChangeSnapshotUnreliableRecoverConcurrentPartition3B(t *testin
 	// Test: unreliable net, restarts, partitions, snapshots, conf change, many clients (3B) ...
 	GenericTest(t, "3B", 5, true, true, true, 100, true, true)
 }
+
+func TestDifferentDBPath(t *testing.T) {
```

Comment:
move to a another place

Reply:
There is no other test file in `/test_raftstore`, but this test is for the `/test_raftstore/cluster.go`, should I create a new test file?(maybe `cluster_test.go`?)

Reply:
`cluster_test.go` lgtm

Reply:
> `cluster_test.go` lgtm

moved